cmd/editaccount.go

```diff
@@ -135,6 +145,8 @@ type EditAccountParams struct {
 	data           NumberParams
 	signingKeys    SigningKeysParams
 	rmSigningKeys  []string
+	r1             JetStreamLimitParams
```

Comment:
I am not sure if this is correct to list only those. I mean technically one can have tiers per replicas count. The server determines if there are tiered limits based on the stream configuration's Replicas count.